ansible/package-installs.yaml

```diff
@@ -1,3 +1,5 @@
 git:
   uninstall: False
-  installtype: package
\ No newline at end of file
+  installtype: package
+ansible:
+python3-openstacksdk:
```

Comment:
Sorry for being VERY late on this topic.
IIRC it felt elegant to have some basic packages in place that most jobs may use.

But this can also be a fallacy based on to less experience the zuul job building.

So you would like to see this file dropped?

Reply:
Yes, in my experience it is better to let the jobs decide what they need.